Spintter/sprintter
# SPR-5 set up database and base service

#### Description

- set up database models
- add .github folder for PR template
- add .env.sample
- add a file to load env vars
- add create-user migration

#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
- Set up a Postgres database
- Add Database URL to .env file
- run `yarn db:migrate`
- Your database should contain a user table
Give instructions on how reviewers can verify your work

#### Any background context you want to add

SPR-5 JIRA

[Finishes SPR-5]
